ambari-web/app/controllers/wizard/step8_controller.js

```diff
@@ -972,6 +971,7 @@ App.WizardStep8Controller = App.WizardStepController.extend(App.AddSecurityConfi
       const serviceGroups = mpacks.map(mpack => ({
           "ServiceGroupInfo": {
             "service_group_name": mpack.name,
+            "version": mpack.id
```

Comment:
shouldn't this be mpack.name + "-" + mpack.version?

Reply:
mpack.id have the same format, and version is taken directly from mpack without UI formatting.